elements-code-tutorial/advanced-examples.md

```diff
@@ -105,11 +105,17 @@ fi
 
 shopt -s expand_aliases
 
-alias e1-dae="$HOME/elements/src/elementsd -datadir=$HOME/elementsdir1"
-alias e1-cli="$HOME/elements/src/elements-cli -datadir=$HOME/elementsdir1"
+# Before running this script, change the following variables to point to: your local Elements binaries directory, the directory you'll use to store node data (note that the script will delete the $DATA_DIR/elementsregtest directory when run, so back up anything you might already have in there before running).
+
+ELEMENTS_DIR="$HOME/elements"
```

Comment:
I think if we're doing this we might as well include the src part as well. I currently use the binary release so I'd have to amend this and line 114 etc if I used this. So add src path to this and then have line 114 and 115 just add the elementsd and elements-cli bits.